Source/Engine/Platform/Linux/LinuxFileSystemWatcher.h

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2012-2021 Wojciech Figat. All rights reserved.
+
+#pragma once
+
+#if PLATFORM_LINUX
+
+#include "Engine/Platform/Base/FileSystemWatcherBase.h"
+#include "../Win32/WindowsMinimal.h"
+
+/// <summary>
+/// Windows platform implementation of the file system watching object.
```

Comment:
This should be Linux instead of windows.

Reply:
Thanks, missed that :)